torch/csrc/profiler/unwind/dwarf_symbolize_enums.h

```diff
@@ -0,0 +1,181 @@
+#pragma once
+#include <torch/csrc/profiler/unwind/unwind_error.h>
+#include <cstdint>
+#include <optional>
+
+enum {
+  DW_TAG_subprogram = 0x2e,
```

Comment:
isn't there some dwarf header we can include that defines all these things? i guess we don't have access to it though?

Reply:
It doesn't come by default. However the numbers themselves are defined by the standard, so they can't change anyway.